etn-ccis/blui-doc-it
# Feature/3748 update themes and calendar

Fixes BLUI-3748.



#### Changes proposed in this Pull Request:

- Update themes to work with mui5 updates



#### To Test:

- clone repo
- run `yarn && yarn start`
- Browse all of the themes (triple-click the header to toggle the global theme easily) to ensure all of the elements affected by holiday themes look as they should, including:
	- graphics
	- drawer
	- FAB
	- banners
	- code blocks
	- custom components (resources, roadmap)
	- etc.
